### PR TITLE
class 18 done

### PR DESCRIPTION
**Pull request by yosef406**
class 18 is done
with some dom styling

### CODE REVIEW

**Comment by yosef406** [comment]
fixed the conflict in readme.md

File: script/script.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    let age = document.getElementById("ageInput");
+    let ID = document.getElementById("idInput");
+    let email = document.getElementById("emailInput");
+
+    let nameTxt = document.getElementById("nameTxt");
+    let ageTxt = document.getElementById("ageTxt");
+    let IDTxt = document.getElementById("idTxt");
+    let emailTxt = document.getElementById("emailTxt");
+
+    let invalidDataText = document.querySelector("h4");
```
Comment (EliasSalom) [inline script/script.js:12]: test

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please check this

File: script/script.js (hunk 1 above)
Comment (yosef406) [reply script/script.js:12]: resolved

**Comment by yosef406** [comment]
closing because its past do.

**Pull request closed without merging**